### PR TITLE
More Ready for Swift 6 progress

### PR DESCRIPTION
**Pull request by daveverwer**
Screenshot 2024-04-29 at 15 39 32@2x

### CODE REVIEW

**Review by finestructure — APPROVED** [review]

**Comment by daveverwer** [comment]
Added a Frequently Asked Questions:

Screenshot 2024-05-01 at 12 01 48@2x

File: Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_ReadyForSwift6Show.1.html [hunk 1]
```diff
@@ -78,16 +78,46 @@ <h1>
...
+        </p>
+        <p>
+          <strong>A: </strong>We define compatibility in the same way we do on package pages. If any build of the package completes successfully on any of our tested platforms (macOS via SwiftPM, macOS via XcodeBuild, iOS, visionOS, watchOS, tvOS, or Linux) then that build is deemed compatible with the Swift version.
+        </p>
+        <hr class="minor"/>
+        <p>
+          <strong>Q: </strong>What does “total concurrency errors” mean?
+        </p>
+        <p>
+          <strong>A: </strong>Sven: Can you replace this with a short description of these errors, please?
```
Comment (daveverwer) [inline Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_ReadyForSwift6Show.1.html:103]: @finestructure I figured you'd do a better job with this description. Could you add it when you have a second please.

File: Sources/App/Views/ReadyForSwift6/ReadyForSwift6Show+View.swift [hunk 2]
```diff
@@ -38,17 +38,74 @@ extension ReadyForSwift6Show {
...
+                .hr(
+                    .class("minor")
+                ),
+                .p(
+                    .strong(.text("Q: ")),
+                    .text("What does “total concurrency errors” mean?")
+                ),
+                .p(
+                    .strong(.text("A: ")),
+                    .text("Sven: Can you replace this with a short description of these errors, please?")
```
Comment (daveverwer) [inline Sources/App/Views/ReadyForSwift6/ReadyForSwift6Show+View.swift:75]: @finestructure I figured you'd do a better job with this description. Could you add it when you have a second please.

Comment (finestructure) [inline Sources/App/Views/ReadyForSwift6/ReadyForSwift6Show+View.swift:75]:
```suggestion
                    .text("Swift 6 introduces complete concurrency checking, which is a compiler feature that checks your code for data-race safety. The number of concurrency errors reflects how many issues the compiler detected relating to these concurrency or data-race checks.")
```

**Pull request merged**